cloud/helm-chart/src/main/helm/zilla/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "zilla.fullname" . }}-test-connection"
+  labels:
+    {{- include "zilla.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "zilla.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
I think this won't work because we don't define one `{{ .Values.service.port }}` right?

Reply:
fixed